README.md

```diff
@@ -33,7 +33,8 @@ CTOP is a tool which allows you to quickly find out what's happening on a machin
 * Metrics are for one node only (not cluster-wide)
 
 ## Notes
-* CTOP should run on anything, but I've only tested it on Ubuntu.
+* CTOP should run on anything. Originally was developed on Ubuntu, runs
+  great on CentOS and MacOS X.
```

Comment:
Can we change this to: "Originally was developed on Ubuntu, and it's reported to run on CentOS and MacOS X"?

Great is relative 8)